signer.go

```diff
@@ -39,60 +42,37 @@ func (s Signer) GenerateToken(message string) string {
 	return s.UrlSafe(digest)
 }
 
-// GenerateFeedScopeToken returns a jwt
-func (s Signer) GenerateFeedScopeToken(context ScopeContext, action ScopeAction, feedIDWithoutColon string) (string, error) {
-
-	claims := jwt.MapClaims{
-		"resource": context.Value(),
-		"action":   action.Value(),
-		// "aud":
-		// "exp": time.Now().UTC().Add(time.Hour * 1),
-		// "jti": uuid.New(),
-		// "iat": time.Now(),
-		// "iss":
-		// "nbf": time.Now().Unix(),
-		// "sub":
-	}
-
-	if feedIDWithoutColon != "" {
-		claims["feed_id"] = feedIDWithoutColon
-	} else {
-		claims["feed_id"] = "*"
+func (s Signer) SignHTTP(request *http.Request) error {
```

Comment:
not strictly related, but shouldn't this be reflected in the REST documentation?